ladybug/epw.py

```diff
@@ -1425,150 +1425,201 @@ class EPWFields(object):
 
         6: {'name': temperature.DryBulbTemperature(),
             'type': float,
-            'unit': 'C'
+            'unit': 'C',
+            'min': -70,
+            'max': 70,
+            'missing': 99.9
             },
 
         7: {'name': temperature.DewPointTemperature(),
             'type': float,
-            'unit': 'C'
+            'unit': 'C',
+            'min': -70,
+            'max': 70,
+            'missing': 99.9
             },
 
         8: {'name': fraction.RelativeHumidity(),
             'type': int,
-            'unit': '%'
+            'unit': '%',
+            'missing': 999,
+            'min': 0,
+            'max': 110
             },
 
         9: {'name': pressure.AtmosphericStationPressure(),
```

Comment:
@chriswmackey, out of curiosity why don't we hardcode the name of each field as a string?

Reply:
I am not sure that I follow your suggestion.  We ultimately need the data type object if we want to create data collections that can be converted to other unit systems and benefit from the other operations the data type objects offer.  Are you saying that we should store the name of the data type as a string and then use an exec() statement to get the data type object of the same name?  Or are you saying that we should have redundancy where the name exists as a sting on both the data type object and in this dictionary for epw fields?